scripts/bundled_installer

```diff
@@ -10,6 +10,9 @@
 export PYTHON_VERSION="3.7.2"
 export PYENV_ROOT="$HOME/.pyenv"
 export PYENV_BIN="$PYENV_ROOT/versions/$PYTHON_VERSION/bin"
+BASH_PROFILE="$HOME/.bash_profile"
+ZSHENV="$HOME/.zshrc"
+PYTHON_INSTALLED_PREVIOUSLY="false"
```

Comment:
Use boolean here. Also, this variable should be called `PYTHON_ALREADY_IN_PATH` because it is set to `true` when `python_is_not_in_path` is `false`